draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -76,6 +84,48 @@ public DraggableView(Context context, AttributeSet attrs, int defStyle) {
   }
 
   /**
+   * Return if user can maximize minimized view on click.
+   */
+  public boolean isClickToMaximizeEnabled() {
+    return enableClickToMaximize;
+  }
+
+  /**
+   * Allow to maximize when the user click on minimized view.
+   */
+  public void setClickToMaximizeEnabled(boolean enableClickToMaximize) {
+    this.enableClickToMaximize = enableClickToMaximize;
+  }
+
+  /**
+   * Return if user can minimize maximized view on click.
+   */
+  public boolean isClickToMinimizeEnabled() {
+    return enableClickToMinimize;
+  }
+
+  /**
+   * Allow to minimize when the user click on maximized view.
```

Comment:
Review this javadoc. This should be something like "Enable or disable click to minimize view when dragged view is maximized".